recipes/peppy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "peppy" %}
+{% set version = "0.30.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 058703fd51eeb5dfb57f0dd2bc24ff11c7d999773f43787d76628138697992aa
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - attmap >=0.12.5
+    - python
+    - pyyaml >=5
+    - ubiquerg >=0.4.9
+    - pandas >=0.24.2
+    - logmuse >=0.2
+
+test:
+  imports:
+    - peppy
+
+about:
+  home: "https://github.com/pepkit/peppy/"
+  license: "BSD-2-Clause"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
```

Comment:
The license file is missing from the source distribution. I've put in a PR to fix this:
https://github.com/pepkit/peppy/pull/354

For now you'll need to manually package the license file:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Also above the `license_file` entry, please put the following comment as a reminder that if the PR is merged, the license file will be available in a future release at which point, the manually packaged copy can be removed:

```
# License file manually. Check the status of https://github.com/pepkit/peppy/pull/354 for future releases
license_file: "LICENSE.txt"
...
```